modules/tinymce/src/core/test/ts/browser/api/EditorCommandsTest.ts

```diff
@@ -321,6 +321,189 @@ describe('browser.tinymce.core.api.EditorCommandsTest', () => {
     });
   });
 
+  context('removeCommand', () => {
+    it('TINY-13144: removeCommand should remove exec command by default', () => {
+      const editor = hook.editor();
+      const state = Singleton.value<CommandTestState<string, Editor>>();
+
+      editor.editorCommands.addCommand('TestRemoveCommand1', (ui, value) => state.set({ ui, value }));
+
+      assert.isTrue(editor.editorCommands.queryCommandSupported('TestRemoveCommand1'), 'Command should be supported before removal');
+      assert.isTrue(editor.editorCommands.execCommand('TestRemoveCommand1', true, 'test'), 'Command should execute successfully before removal');
+      assert.deepEqual(state.get().getOrDie('Should exist a state'), { ui: true, value: 'test' });
+
+      state.clear();
+      editor.editorCommands.removeCommand('TestRemoveCommand1');
+
+      // Verify command is removed
```

Comment:
Are some of these comments stating the obvious the assert messages is clear I think. Some of them might be tweaked a little to be more clear.